api/README.md

```diff
@@ -1,4 +1,4 @@
-This directory contains code pertaining to the Docker API:
+This directory contains code pertaining to the [Docker API](https://docker.github.io/engine/reference/api/docker_remote_api/):
```

Comment:
@thaJeztah Alright! 
But we can update it here also as it's a part of Docker Docs!
Should I update it here itself or not?